logging-modules/log4j/src/main/java/com/baeldung/reactiveLogging/WebFluxLogging.java

```diff
@@ -0,0 +1,26 @@
+package com.baeldung.reactiveLogging;
+
+import reactor.core.publisher.Flux;
+
+public class WebFluxLogging {
```

Comment:
Can you rename this to `WebFluxLoggingExample` as we have done with the other files in this folder?

Reply:
This is fixed.